### PR TITLE
Pensions employment history question

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3489

image

image

### CODE REVIEW

**Comment by cvalarida** [comment]
I was looking for the `employmentHistory` unit test and didn't find one, but I did find that `powAndSeverance` wasn't a `.unit.spec`, so I renamed it while I was here.

**Comment by jbalboni** [comment]
@cvalarida That description in jsx in the helpers file, so you could add a class to the element and add some margin.

File: test/pensions/config/employmentHistory.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          onSubmit={onSubmit}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = getFormDOM(form);
+    formDOM.setYesNo('#root_view\\:workedBeforeDisabledYes', true);
```
Comment (jbalboni) [inline test/pensions/config/employmentHistory.unit.spec.jsx]: This doesn't seem notably better than `formDOM.fillData('#root_view\\:workedBeforeDisabledYes', 'Y');`

File: test/pensions/config/employmentHistory.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = getFormDOM(form);
+
+    formDOM.setYesNo('#root_view\\:workedBeforeDisabledYes', true);
+
+    expect(formDOM.querySelectorAll('input,select').length).to.equal(18);
+  });
+
+  it('should not submit empty form', () => {
```
Comment (jbalboni) [inline test/pensions/config/employmentHistory.unit.spec.jsx]: Can you add a test to make sure you can submit if you choose No to the first question?

File: yarn.lock [hunk 3]
```diff
@@ -7787,7 +7787,7 @@ supports-color@1.2.0:
   version "1.2.0"
   resolved "https://registry.yarnpkg.com/supports-color/-/supports-color-1.2.0.tgz#ff1ed1e61169d06b3cf2d588e188b18d8847e17e"
 
-supports-color@3.1.2, supports-color@^3.1.0, supports-color@^3.1.1:
```
Comment (jbalboni) [inline yarn.lock]: Doesn't seem like we need these lock file changes.

File: test/burials/config/employmentHistory.unit.spec.jsx [hunk 4]
```diff
@@ -1,153 +0,0 @@
-import React from 'react';
```
Comment (jbalboni) [inline test/burials/config/employmentHistory.unit.spec.jsx]: lol, oops

**Review by jbalboni — APPROVED** [review]
Looks good. Made a comment about adding another test and fixing spacing.

File: yarn.lock (hunk 3 above)
Comment (cvalarida) [inline yarn.lock]: Yeah, I don't know what the deal is with these. I had to `yarn install` to get the correct version of vets-json-schema, but that also updated the lockfile...Usually I just ignore it, but...I guess I didn't this time. I'll change that back.

File: test/pensions/config/employmentHistory.unit.spec.jsx (hunk 1 above)
Comment (cvalarida) [inline test/pensions/config/employmentHistory.unit.spec.jsx]: Not really, no, but I made the ability for it to accept either `'Y'` or `true`.

**Comment by cvalarida** [comment]
Updated the spacing.

**Pull request merged**